editor/file_system/editor_file_system.h

```diff
@@ -246,6 +247,16 @@ class EditorFileSystem : public Node {
 	void _save_filesystem_cache();
 	void _save_filesystem_cache(EditorFileSystemDirectory *p_dir, Ref<FileAccess> p_file);
 
+	static bool _has_editor_variant_import(const String &p_path);
+	static bool _is_test_for_editor_resources_reimport_needed();
+	static String _editor_import_meta_path();
+	static void _load_editor_import_meta(Dictionary *p_values);
```

Comment:
```suggestion
	static void _load_editor_import_meta(Dictionary &r_values);
```
Generally there's no reason to pass by pointer unless the value is allowed to be null, if it is there should be checks for it